### PR TITLE
Add language about experimental hugo support

### PR DESCRIPTION
**Pull request by jseppi**
Also fixed some minor spelling and markdown-linting-related stuff.

Related: https://github.com/18F/federalist-garden-build/pull/64

### CODE REVIEW

File: pages/using-federalist.md [hunk 1]
```diff
@@ -13,15 +13,18 @@ After you pick a template, your new site will be built and added to your list of
...
 #### Multi Page Template
+
 ![Multi Page Template](https://federalist.18f.gov/images/team.thumb.png)
 
 #### Single Page Template
+
 ![Single Page Template](https://federalist.18f.gov/images/landing.thumb.png)
 
-Additionally, Federalist will build any Jekyll-based website, supporting [custom website templates]({{site.baseurl}}/pages/custom-templates/).
+Additionally, Federalist will build any Jekyll or Hugo website, supporting [custom website templates]({{site.baseurl}}/pages/custom-templates/).
```
Comment (wslack) [inline pages/using-federalist.md]: can we add a note that this is experimental here?

File: pages/using-federalist.md [hunk 2]
```diff
@@ -31,14 +34,13 @@ The [18F method cards](https://methods.18f.gov/) are an overview of several diff
...
 
-
 ## Managing site settings
 
 The settings option for a site lets you adjust how Federalist handles the site. These settings are for advanced functionality and most people will not need to change them.
 
 - **Default branch** This is the branch of the site's repository that is built and hosted as the production version of the site. Other branches are considered drafts. Changing this value will rebuild the site using a different branch.
 
-- **Static Site Engine** This controls if Federalist builds your site using Jekyll then hosts it, or simply hosts content you've already generated. If using a template, this should always be "Jekyll."
+- **Static Site Engine** This controls if Federalist builds your site using Jekyll or Hugo, or simply hosts content you've already generated. If using a template, this should always be "Jekyll."
```
Comment (wslack) [inline pages/using-federalist.md]: I wonder if this is too easy to get into debt with; maybe say "This controls how Federalist technically builds your site and should not be altered once your site is live. Contact the Federalist team for additional support."

Thoughts?

Comment (jseppi) [inline pages/using-federalist.md]: Sure, though that makes me think we should change the interface to make the site engine dropdown read-only after a site is initially added. 🤔

Comment (wslack) [inline pages/using-federalist.md]: we've had to make a change in that dropdown before but I wonder if it should be down with the "delete a site" button in the red box.

**Pull request merged**